main/zoe/api/records.md

```diff
@@ -5,27 +5,30 @@
 These records (JavaScript objects with data properties and no
 functions) are used throughout Zoe.
 
-## SeatAndPayout
-
-This is returned by a call to `redeem` on Zoe. A seat is an arbitrary
-object whose methods allow the user to take certain actions in a
-contract. The payout is a promise that resolves to an object which has
-keywords as keys and promises for payments as values. Keywords are
-strings that must begin with a capital letter and must be ASCII.
-Parties to the contract will use the keywords to index their proposal
-and their payments.  Note that while the payout promise resolves when
-an offer is completed, the promise for each payment resolves after the
-remote issuer successfully withdraws the payment.
+## OfferResultRecord
+
+This is returned in a promise by a call to `offer` on Zoe. It contains 
+a handle for querying Zoe about the offer, a promise for the payouts 
+when the offer is complete, a promise for the result of invoking the contract-specific hook associated with the invitation, and if 
+appropriate for the specified `exit` policy, a function to cancel 
+the offer.
 
 ```js
 {
-  seat: someSeat,
-  payout: paymentKeywordRecord,
+  offerHandle: someSeat,
```

Comment:
```suggestion
  offerHandle: Promise<Handle>,
  outcome: Promise<Any>,
  payout: Promise<PaymentKeywordRecord>
  cancelObj,
```
This part doesn't make sense to me. An offerHandle is the identifier for an offer. There are no seats anymore.

Reply:
I missed editing "someSeat". From the other examples, ti seemed like this wasn't doing the argument types, but rather some vaguely related name. But from a type point of vie, isn't that the presence and not the promise for a handle?

Reply:
No, it's specifically a promise for a handle

Reply:
```
            const offerResult = {
              offerHandle: HandledPromise.resolve(offerHandle),
              payout: payoutMap.get(offerHandle).promise,
              outcome: outcomeP,
            };
```

Reply:
I rewrote some that with names rather than types, but tried to do so in a way that matched your change.